mozilla/blurts-server
# Mntor 1254/update layouts

# References: 
Jira: MNTOR-1254




# Description
The Landing, error, email preview, and all other pages that should be displayed using guest/anonymous layout – no nav, avatar, user menu, etc.  Currently they overload the "main" (signed-in user) layout using conditional and irrelevant code.  With a separate and specific layout we can separate concerns, de-clutter code, and help avoid confusion.


# Screenshot (if applicable)

Not applicable.

# How to test
- visit guest/anonymous pages to confirm they display correctly and do not display nav, user menu, etc
- visit logged-in-user pages to confirm they show nav, user menu, etc.
